### PR TITLE
Grammar Update to organizations.md

### PR DESCRIPTION
**Pull request by mattou07**
## Description

_What did you add/update/change?_
https://docs.umbraco.com/umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations

## Type of suggestion

* [x] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Cloud

## Deadline (if relevant)

_When should the content be published?_

No deadline.

### CODE REVIEW

**Comment by jonat123** [comment]
@mattou07 Thanks for the PR 😊

I noticed that you are introducing some vale warnings about too-long sentences that have been introduced with your changes 🙈

Could you take a look at them and make sure that the sentences are not longer than 25 words?

**Comment by jonat123** [comment]
@mattou07 I was wondering, whether you have seen my previous message? 😊

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Organizations
 
-On Umbraco Cloud it is possible to get an Organization. An organization is handy if you are a partner or are managing a lot of projects for different customers.
+On Umbraco Cloud it is possible to setup an Organization. An organization is handy if you need to manage permissions for multiple people (such as developers to content editors) or are managing a lot of projects for different customers.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:3]:
```suggestion
On Umbraco Cloud it is possible to setup an Organization. An organization is handy if you are managing many projects for different customers. It is also handy if you need to manage permissions for multiple users (such as developers, content editors etc.). 
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-With an organization, you get an overview of all projects and members that are part of it, manage the project payments, and many others.
+With an organization, you get an overview of all projects and members that are part of it, manage payment methods for projects, as well as many other functions outlined on this page.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:5]:
```suggestion
With an organization, you get an overview of all projects and members that are part of it. You can also manage payment methods for projects, as well as many other functions outlined on this page.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 3]
```diff
@@ -17,17 +17,17 @@ Are you interested in getting an organization, or need a project added to a diff
 
 ### Information
 
-In the **Information** section of the Organization, you can find information about your Organization. If there are any changes to your Information, you can change them from this section.
+In the **Information** section of the Organization, you can find all your details about your Organization. If there are any changes to your detais, you can change them here.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:20]:
```suggestion
In the **Information** section of the Organization, you can find all the details about your Organization. If there are any changes to your details, you can change them here.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 4]
```diff
@@ -62,12 +62,12 @@ Organization Members with **Read Access**, can do the following in the organizat
...
 
 {% hint style="info" %}
-Being a Member of an organization, does not give access to any projects under it. To get access to a project under an organization, you need to be [invited](/umbraco-cloud/set-up/users-on-cloud.md) to the project. This can be done by either someone that is part of the project or an administrator in your organization.
+Being a Member of an organization, does not give access to any projects under it. To get access to a project under an organization, you need to be [invited](/umbraco-cloud/set-up/users-on-cloud.md) to the project. This can be done by either someone that is already part of the project or an administrator in your organization.
 {% endhint %}
 
 ### Multi Factor Authentication (MFA) enforcement
 
-When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of MFA for members.
+When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of MFA method for members.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:70]:
```suggestion
When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of Multi-Factor Authentication (MFA) method for members.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 5]
```diff
@@ -81,7 +81,7 @@ To enforce a certain MFA for a member, follow these steps:
 4. Find the member that needs to have MFA enabled.
 5. Click on the cogwheel and select the **Enforced MFA Method** from the drop-down list for the member.
 
-Once it has been enabled, the next time the member logs in, they will be enforced to set up the MFA that has been chosen.
+Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen MFA method.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:84]:
```suggestion
Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen Multi-Factor Authentication (MFA) method.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 6]
```diff
@@ -17,17 +17,17 @@ Are you interested in getting an organization, or need a project added to a diff
...
 
-In the **Information** section of the Organization, you can find information about your Organization. If there are any changes to your Information, you can change them from this section.
+In the **Information** section of the Organization, you can find all your details about your Organization. If there are any changes to your detais, you can change them here.
 
 ![Information](images/Information-v10.png)
 
 ### Members
 
-In the **Members** section, you can get an overview, view pending invites, and the Multi-Factor Authentication (MFA) for the Members of your Organization. It is also possible to set up different permissions for your Members, such as Read, Write, and Administrators for your organization.
+In the **Members** section, you can view current members, view pending invites, and see the Multi-Factor Authentication (MFA) status for the Members of your Organization. You can also set up different permissions for your Members, such as Read, Write, and Administrators for your organization by adjusting their **Role**.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:26]:
```suggestion
In the **Members** section, you can view current members, pending invites, and see the Multi-Factor Authentication (MFA) status for the Members of your Organization. You can also setup different permissions for your Members, such as Read, Write, and Administrators for your organization by adjusting their **Role**.
```

**Review by alina-tincas — COMMENTED** [review]
making the necessary changes

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 7]
```diff
@@ -81,7 +81,7 @@ To enforce a certain MFA for a member, follow these steps:
 4. Find the member that needs to have MFA enabled.
 5. Click on the cogwheel and select the **Enforced MFA Method** from the drop-down list for the member.
 
-Once it has been enabled, the next time the member logs in, they will be enforced to set up the MFA that has been chosen.
+Once it has been enabled, the next time the member logs in, they will be forced to set up the chosen Multi-Factor Authentication (MFA) method.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:84]:
```suggestion
Once it has been enabled, the next time the member logs in, they will be forced to setup the chosen Multi-Factor Authentication (MFA) method.
```

File: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md [hunk 8]
```diff
@@ -62,12 +62,12 @@ Organization Members with **Read Access**, can do the following in the organizat
...
 
 {% hint style="info" %}
-Being a Member of an organization, does not give access to any projects under it. To get access to a project under an organization, you need to be [invited](/umbraco-cloud/set-up/users-on-cloud.md) to the project. This can be done by either someone that is part of the project or an administrator in your organization.
+Being a Member of an organization, does not give access to any projects under it. To get access to a project under an organization, you need to be [invited](/umbraco-cloud/set-up/users-on-cloud.md) to the project. This can be done by either someone that is already part of the project or an administrator in your organization.
 {% endhint %}
 
 ### Multi Factor Authentication (MFA) enforcement
 
-When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of MFA for members.
+When working in organizations on Umbraco Cloud, as a company, you might want to enforce a certain type of Multi-Factor Authentication (MFA) method for members.
```
Comment (alina-tincas) [inline umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md:70]:
```suggestion
When working in organizations on Umbraco Cloud, as a company, you can enforce a certain type of Multi-Factor Authentication (MFA) method for members.
```

**Review by alina-tincas — APPROVED** [review]
fixing lint errors

**Pull request merged**